addons/udes_stock/models/stock_warehouse.py

```diff
@@ -78,6 +78,8 @@ def _domain_warehouse_picking_types(self):
                                          default='^PAL(\\d)+$')
     u_package_barcode_regex = fields.Char('Package Barcode Format',
                                           default='^PACK(\\d)+$')
+    u_product_barcode_regex = fields.Char('Product Barcode Replacement Regex',
```

Comment:
Could you update the README for this new field as well

Reply:
Will do

Reply:
Done